test/autotest.py

```
@@ -1475,10 +1480,44 @@ def _categorize_tests(cls, tests: Iterable[TestSpec]) -> List[TestSpec]:
             for test in tests
         ]
 
+    @staticmethod
+    def _clang_runtime_dir() -> str:
+        # clang's -shared-libsan TSAN runtime lives in a non-standard dir with no
+        # RPATH, so the tsan-clang test needs LD_LIBRARY_PATH pointing at it.
+        try:
+            out = subprocess.run(["clang", "-print-runtime-dir"],
```

Ruff (0.15.18)

[error] Starting a process with a partial executable path

(S607)